src/main/resources/db/changelog-master.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <include id="Initial script" file="init.sql" relativeToChangelogFile="true"/>
```

Comment:
The `id` attribute can be redundant.

 What does the `relativeToChangelogFile` attribute mean?

Reply:
The migration will not work without the `relativeToChangelogFile` since Liquibase will try to search the `init.sql` file in the Spring classpath. When the `relativeToChangelogFile` is set to `true` - Liquibase will be looking for the `init.sql` in the same directory where the `changelog-master.xml` is located.

Otherwise, we need to set the `file` attribute as `classpath:/db/init.sql` which seems less readable in my opinion.